enterprise/frontend/src/metabase-enterprise/content_verification/index.ts

```diff
@@ -0,0 +1,7 @@
+import { PLUGIN_CONTENT_VERIFICATION } from "metabase/plugins";
+import { hasPremiumFeature } from "metabase-enterprise/settings";
+import { VerifiedFilter } from "metabase-enterprise/content_verification/VerifiedFilter/VerifiedFilter";
```

Comment:
nit: what do you think about adding `.../VerifiedFilter/index.ts` so we don't need to see `VerifiedFilter` being repeated when importing the component?